src/index.html

```diff
@@ -10,23 +10,34 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar airbnb</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search__form search__icon--size--l"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        placeholder='Try "Los Angeles"'
```

Comment:
Hello, @maristella12!

Here you really need to use `placeholder="Try “Los Angeles“"` as your placeholder. You can copy and paste it. Using single quotes is still crashing the tests.

The quotes on Los Angeles is a special character, please adjust using that.